test/OpenTelemetry.Tests/SdkOptionsTests.cs

```diff
@@ -17,40 +17,38 @@
 using System;
 using Xunit;
 
-namespace OpenTelemetry.Configuration.Tests
+namespace OpenTelemetry.Tests
 {
-    [Collection("xUnitCollectionPreventingTestsThatDependOnSdkConfigurationFromRunningInParallel")]
-    public class SdkConfigurationTests : IDisposable
+    [Collection("xUnitCollectionPreventingTestsThatDependOnSdkOptionsFromRunningInParallel")]
```

Comment:
Probably unnecessary now that the static Instance is gone.

Reply:
Removed